### PR TITLE
Adds a brief line on usage to the readme

### PR DESCRIPTION
**Pull request by eddiebergman**
I'm not much of a video watcher for tutorials but having a quick section on how it's used might be nice :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ Gboard Dictionary for easily typing unicode symbols with shortcuts based on LaTe
 
 Shortcuts for Gboard are supported on all Android devices. As of now, these shortcuts cannot be imported to iOS devices.
 
+## Usage
+It's as simple as typing \R or \sum on the keyboard and tapping the suggested symbol!
```
Comment (DenverCoder1) [inline README.md:13]:
```suggestion

It's as simple as typing `\R` or `\sum` on the keyboard and tapping the suggested symbol!
```

**Review by DenverCoder1 — CHANGES_REQUESTED** [review]
Looks good.

Maybe a GIF could help demonstrate it as well.

**Comment by eddiebergman** [comment]
I'm not familiar with phone recording as you did with the video, I could possible add a screenshot but if you'd rather motion with a gif then I unfortunately don't have the time to figure it all out.

Nice little hack for symbols by the way, appreciated :)

**Review by DenverCoder1 — APPROVED** [review]
Looks good!

Animations and stuff could always be added at a later time.

**Pull request merged**